guides/common/modules/proc_upgrading-hosts-to-next-major-rhel-release.adoc

```diff
@@ -36,9 +36,16 @@ For more information, see xref:Distributing_SSH_Keys_for_Remote_Execution_{conte
 . Click *Submit* to start the pre-upgrade check.
 . When the check is finished, click the *Leapp preupgrade report* tab to see if Leapp has found any issues on your hosts.
 Issues that have the *Inhibitor* flag are considered crucial and are likely to break the upgrade procedure.
-Some issues might have documentation linked that describes how to fix them.
-. Optional: If you have issues that have commands associated with them, you can fix them using remote execution.
-To do that, select these issues, click the *Fix Selected* button, and submit the job.
+Issues that have the *Has Remediation* flag contain remediation that can help you fix the issue.
+. Click an issue that is flagged as *Has Remediation* to expand it.
```

Comment:
To clarify, I meant the entire section about issues should be one step because you do not fix issue if you have none. Thus they should be contained within a single step. What do you think?

```suggestion
Click an issue that is flagged as *Has Remediation* to expand it.
```

Reply:
In that case it should include the following step as well. And there might be substeps. I'll try something.